react/form/src/auth/useAuthProvider.js

```diff
@@ -0,0 +1,16 @@
+import React, { Fragment } from 'react';
+import WidgetKeycloakProvider from 'auth/keycloak/WidgetKeycloakProvider';
+import useKeycloak from 'auth/keycloak/useKeycloak';
+
+const useAuthProvider = WrappedComponent => {
+  return props => {
+    const Provider = useKeycloak ? WidgetKeycloakProvider : Fragment;
```

Comment:
I find this approach somehow misleading. It uses the `use` prefix like a hook, it works like a HOC but it's neither a hook nor a HOC.  I think it could be better to use this ternary operator in the WebComponent render method, alongside other providers like theme provider, style provider, etc.